lib/rouge/lexers/dart.rb

```diff
@@ -61,7 +61,8 @@ class Dart < RegexLexer
 
         rule %r/#{id}:/, Name::Label
         rule %r/\$?#{id}/, Name
-        rule %r/[~^*!%&\[\](){}<>\|+=:;,.\/?-]/, Operator
+        rule %r/[~^*!%&\|+=:;,.\/?-]/, Operator
+        rule %r/[\[\](){}<>]/, Punctuation
```

Comment:
**suggestion:** Should `,.;` be punctuations? I am not familiar with Dart and just refer to [Pygments](https://github.com/pygments/pygments/blob/64e8e05307689fe726ce9df2d5907e9c2fb67405/pygments/lexers/javascript.py#L462).

Reply:
Thanks @tancnle you're right I've updated it